website/pages/en/quick-start.mdx

```diff
@@ -2,24 +2,20 @@
 title: Quick Start
 ---
 
-This guide will quickly take you through how to initialize, create, and deploy your subgraph to Subgraph Studio.
+This guide walks you through the basics of creating, deploying, publishing, and querying a subgraph on The Graph easily.
 
-Ensure that your subgraph will be indexing data from a [supported network](/developing/supported-networks).
-
-This guide is written assuming that you have:
+**Prerequisites for this guide:**
 
 - A crypto wallet
 - A smart contract address on the network of your choice
+- A subgraph that will be indexing data from a [supported network](/developing/supported-networks/)
+- An API key
 
-## 1. Create a subgraph on Subgraph Studio
-
-Go to the [Subgraph Studio](https://thegraph.com/studio/) and connect your wallet.
+## Step-by-Step
 
-Once your wallet is connected, you can begin by clicking “Create a Subgraph." It is recommended to name the subgraph in Title Case: "Subgraph Name Chain Name."
+### 1. Install the Graph CLI
 
-## 2. Install the Graph CLI
-
-The Graph CLI is written in TypeScript and you will need to have `node` and either `npm` or `yarn` installed to use it. Check that you have the most recent CLI version installed.
+The Graph CLI is written in TypeScript, and you must have `node` and either `npm` or `yarn` installed to use it. Check for the most recent CLI version.
```

Comment:
Let's avoid technical detail of what CLI is written in and stay to the point that they need to have `node` installed
 
```suggestion
You must have [Node.js](https://nodejs.org/en) and package manager of your choice (`npm`, `yarn` or `pnpm`)  installed to use the Graph CLI. Check for the [most recent](https://github.com/graphprotocol/graph-tooling/releases?q=%40graphprotocol%2Fgraph-cli&expanded=true) CLI version.
```